### PR TITLE
Create RPCs and messages to support new views

### PR DESCRIPTION
**Pull request by dylanratcliffe**
These RPCs and messages will make the views that have currently been designed in Figma a hell of a lot smoother as they return (within reason) only the data that is actually required, and some combine other RPCs into a single call

## Implementation tickets

* https://github.com/overmindtech/api-server/issues/213
* https://github.com/overmindtech/api-server/issues/214
* https://github.com/overmindtech/api-server/issues/215
* https://github.com/overmindtech/api-server/issues/216
* https://github.com/overmindtech/api-server/issues/217
* https://github.com/overmindtech/api-server/issues/218
* https://github.com/overmindtech/api-server/issues/219
* https://github.com/overmindtech/api-server/issues/220
* https://github.com/overmindtech/api-server/issues/221
* https://github.com/overmindtech/api-server/issues/222

### CODE REVIEW

File: items.proto [hunk 1]
```diff
@@ -190,20 +191,14 @@ message Query {
...
-  string responseSubject = 17;
-
-  // Subject that errors will be sent to
-  string errorSubject = 18;
+message QueryResponse {
+  oneof response_type {
+    Item newItem = 2; // A new item that has been discovered
+    Response response = 3; // Status update
+    QueryError error = 4; // An error has been encountered
+  }
```
Comment (DavidS-om) [inline items.proto:201]: Please remove this change as it would be breaking a lot of unrelated stuff right now.

File: gateway.proto [hunk 2]
```diff
@@ -56,6 +56,8 @@ message GatewayRequest {
         StoreBookmark storeBookmark = 14;
         // load and execute a bookmark into the current state
         LoadBookmark loadBookmark = 15;
+
+        // BlastRadiusBookmark calculateBlastRadius = 16;
```
Comment (DavidS-om) [inline gateway.proto:60]: Remove obsolete design sketch

File: changes.proto [hunk 3]
```diff
@@ -124,8 +338,9 @@ message AppProperties {
   // Example: "Contoso Manager"
   string name = 1;
 
-  // Description of this App
-  // Example: "The Contoso Manager manages the contoso. Notable components are storage, compute and the web. Talk to Jimmy or Susan on 555-CM LEAD for more details."
+  // Description of this App Example: "The Contoso Manager manages the contoso.
+  // Notable components are storage, compute and the web. Talk to Jimmy or Susan
+  // on 555-CM LEAD for more details."
```
Comment (DavidS-om) [inline changes.proto:343]:
```suggestion
  // Description of this App. Example: "The Contoso Manager manages the contoso.
  // Notable components are storage, compute and the web. Talk to Jimmy or Susan
  // on 555-CM LEAD for more details."
```

**Review by DavidS-om — CHANGES_REQUESTED** [review]
See comment

**Review by getinnocuous — APPROVED** [review]
LGTM

**Review by DavidS-om — APPROVED** [review]
ship it!
